src/Command/Import/ImportCommand.php

```diff
@@ -61,7 +63,8 @@ public function __construct(
         WallabagV1Import $wallabagV1Import,
         ElcuratorImport $elcuratorImport,
         ShaarliImport $shaarliImport,
-        PocketHtmlImport $pocketHtmlImport
+        PocketHtmlImport $pocketHtmlImport,
+        OmnivoreImport $omnivoreImport
```

Comment:
nitpick: would have added it after DeliciousImport, like the property and initialization in constructor, and like it's done in 2.6 😅